lib/db/mem.js

```diff
@@ -104,6 +112,11 @@ module.exports = function (log, error) {
       lastAccessTime: sessionToken.createdAt
     }
 
+    unverifiedTokens[tokenId] = {
+      tokenVerificationId: sessionToken.tokenVerificationId,
+      uid: sessionToken.uid.toString('hex')
```

Comment:
Fixed in https://github.com/mozilla/fxa-auth-db-mysql/pull/132/commits/cdc18d499aa88b5e0238d1a5b88fc6e0e5d9a61b.